network/mode/bridge/bridge.go

```diff
@@ -38,62 +36,33 @@ func New(ctx context.Context, config network.BridgeConfig, manager mgr.NetworkMg
 	}
 
 	// get bridge ip
-	bridgeIP := utils.StringDefault(config.IP, DefaultIPNet)
-	ipNet, err := netlink.ParseIPNet(bridgeIP)
+	bridgeIP := utils.StringDefault(config.IPv4, DefaultIPv4Net)
+	ipv4Net, err := netlink.ParseIPNet(bridgeIP)
 	if err != nil {
 		return fmt.Errorf("failed to parse ip %v", bridgeIP)
 	}
-	logrus.Debugf("initialize bridge network, bridge ip: %s.", ipNet)
+	logrus.Debugf("initialize bridge network, bridge ip: %s.", ipv4Net)
 
 	// init host bridge network.
-	_, err = initBridgeDevice(bridgeName, ipNet)
+	_, err = initBridgeDevice(bridgeName, ipv4Net)
```

Comment:
@rudyfly 

Bridge device already supported IPv6 since it deferred the IPv6 allocation by libnetwork IPAM module. So I think that there is no need to add assign bridge IPv6 address in here. WDYT ?

Intergration-test might be blocked by other test suite. So I'll work on the integration-test if upstream integration-test is ready.

Currently could you help to merge the PR to alibaba/libnetwork firstly? then I can vendor the new commit to build the integration-test.

For the failure of the upstream pouch integration-test. I also Cc'ed @allencloud
Steps: 
```shell 
$mkdir -p $GOPATH/src/github.com/alibaba && git clone https://github.com/alibaba/pouch
$cd $GOPATH/src/github.com/alibaba/pouch && make build-integration-test
$make integration-test
integration-test
./hack/testing/run_daemon_integration.sh
stop local-persist volume plugin...
start local-persist volume plugin...
stop pouch daemon...
start pouch daemon...
failed to start pouch daemon in background!
make: *** [integration-test] Error 1
```